### ISSUE TITLE
openpifpaf.predict returns empty list

### ISSUE THREAD

**Issue by PyroX2** [openpifpaf/openpifpaf#621]
Hello, I'm trying to train openpifpaf neural network using custom dataset. Right now just for testing I set up COCO dataset with nose keypoint deleted just to test it for different amount of keypoints than the default. As suggested in the documentation. firstly I'm trying to overfit just on a single image. For this purpose I created a new .json file with annotations and new directory that just contains this one image. Then I trained the neural network and it seemed correctly. Loss decreased from around 2000 to -140. However when I use openpifpaf.predict on the same image as I trained the network it returns just empty list and the output image is the same as input image. Do you know what might be the problem here?
Moreover using openpifpaf.predict with shufflenetv2k16 I get the correct result for pretrained network (with 17 keypoints). However when I use pretrained shufflenetv2k16-apollo-66 I also doesn't get any predictions.

**Comment by PyroX2** [comment]
I found that the problem is in cifcaf.py in __call__ method when generating annotations. At that point self.cpp_decoder.call_with_initial_annotations receives fields with size [16, 5, 28, 41] and returns empty list. I'm not sure why this happens at the moment
